vitrina/cms/models.py

```diff
@@ -133,17 +153,19 @@ class LearningMaterial(models.Model):
     topic = models.CharField(max_length=255, blank=True, null=True)
     user = models.ForeignKey(User, models.DO_NOTHING, blank=True, null=True)
     video_url = models.CharField(max_length=255, blank=True, null=True)
-    imageuuid = models.CharField(max_length=36, blank=True, null=True)
+    image = models.ImageField(max_length=255, blank=True, null=True, upload_to='image/')
```

Comment:
All images should go fo `files/{uuid}` and the field should be filer image field.